### PR TITLE
mkdir: reword examples

### PR DESCRIPTION
**Pull request by SethFalco**
Signed-off-by: Seth Falco <seth@falco.fun>

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):*

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages/windows/mkdir.md [hunk 1]
```diff
@@ -9,4 +9,4 @@
 
 - Recursively create a nested directory tree:
```
Comment (navarroaxel) [inline pages/windows/mkdir.md:10]: Could we start with verb?
```suggestion
- Create a nested directory tree recursively:
```

File: pages/windows/mkdir.md [hunk 2]
```diff
@@ -7,6 +7,6 @@
 
 `mkdir {{directory_name}}`
 
-- Recursively create a nested directory tree:
+- Create a nested directory tree recursively:
 
-`mkdir {{path/to/sub_directory_name}}`
+`mkdir --parents {{path/to/sub_directory_name}}`
```
Comment (navarroaxel) [inline pages/windows/mkdir.md:12]: I tested `mkdir` in WIndows and this example creates 2 paths, a directory named `--parents`, and `path/to/dir`.
`mkdir` doesn't accept argument, and `mkdir --help` creates a `--help` directory. The directory tree is created without need of an extra argument. so:
```suggestion
`mkdir {{path/to/sub_directory}}`
```

File: pages/windows/mkdir.md [hunk 3]
```diff
@@ -7,6 +7,6 @@
 
 `mkdir {{directory_name}}`
```
Comment (navarroaxel) [inline pages/windows/mkdir.md:8]:
```suggestion
`mkdir {{directory}}`
```

File: pages/windows/mkdir.md (hunk 2 above)
Comment (SethFalco) [reply pages/windows/mkdir.md:12]: Ahh shoot… I'm an idiot.
I thought I saw this in the common directory. :facepalm:

**Comment by SethFalco** [comment]
I probably should've closed and reopened a new PR since now we've entirely changed the scope of it I guess. ^-^'

Regardless, this is no longer about the `-p` argument which is on Linux systems only.
Rather it's just got the proposed rewording suggested by other maintainers.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**